plugins/system/cookiemanager/tmpl/default.php

```diff
@@ -13,11 +13,15 @@
 use Joomla\CMS\Language\Text;
 
 $sitemenu = $this->app->getMenu();
-$menuitem = $sitemenu->getItem($params->get('policylink', '101'));
+$menuitem = $sitemenu->getItem($params->get('policylink', ''));
 
-$consentBannerBody = '<p>' . Text::_('COM_COOKIEMANAGER_COOKIE_BANNER_DESCRIPTION') . '</p><p><a '
-	. ' href="' . $menuitem->link . '">' . Text::_('COM_COOKIEMANAGER_VIEW_COOKIE_POLICY') . '</a></p>'
-	. '<h5>' . Text::_('COM_COOKIEMANAGER_MANAGE_CONSENT_PREFERENCES') . '</h5><ul>';
+$consentBannerBody = '<p>' . Text::_('COM_COOKIEMANAGER_COOKIE_BANNER_DESCRIPTION') . '</p>';
+
+if ($menuitem) {
+	$consentBannerBody .= '<p><a  href="' . $menuitem->link . '">' . Text::_('COM_COOKIEMANAGER_VIEW_COOKIE_POLICY') . '</a></p>';

```

Comment:
@shazmasiddiqui Let's use the HTMLHelper here for the link like this:
`HTMLHelper::_('link', $menuitem_link, Text::_('COM_COOKIEMANAGER_VIEW_COOKIE_POLICY'));`